src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -219,7 +219,7 @@ public Videobridge()
      */
     public @NotNull Conference createConference(Jid focus, Localpart name, String gid)
     {
-        return this.createConference(focus, name, /* enableLogging */ true, gid);
+        return this.createConference(focus, name == null ? null : name.toString(), /* enableLogging */ true, gid);
```

Comment:
Nit:  I think we can push this back one more level (`VideobridgeShim`) and simplify to `Objects.toString(name)`